index.html

```diff
@@ -173,6 +173,38 @@ <h3>Research</h3>
         </div>
       </div>
     </div>
+      
+    <div class="registration">
+      <div class="container">
+        <!-- Header -->
+        <div class="registration-header">
+          <h1>Registration</h1>
+        </div>
+        <!-- Body -->
+        <div class="text-center">
+          <div class="row">
+            <div class="col-sm">
+              <div class="hacker">
+                <p>Come build using hardware, software, or anything you want with other hackers! You don't have to know how to code to be a hacker. Hackers can be students, designers, or just any tech-lover out there!</p>
+              </div>
+              <button class="btn">Hacker</button>
```

Comment:
question: does the button text too bold ? i feel like they should be slightly thinner to be closer to wireframe, but couldn't make subtle change with css

Reply:
i think it looks ok!